javascript-modal/main.js

```diff
@@ -0,0 +1,17 @@
+var $overlay = document.querySelector('.overlay');
+var $openModalButton = document.querySelector('.open-modal');
+var $closeModalButton = document.querySelector('.close-modal');
+
+$openModalButton.addEventListener('click', toggleModal);
+$closeModalButton.addEventListener('click', toggleModal);
+
+var modalSwitch = true;
+
+function toggleModal(event) {
+  modalSwitch = !modalSwitch;
+  if (modalSwitch === true) {
+    $overlay.style.visibility = 'hidden';
```

Comment:
This becomes inline styling, which is a `no-no`. Instead, you should create a class you can add and remove.

Additionally, you should use `display: none` rather than `visibility: hidden`, since `visibility` makes the element still take up space, even if it is not visible.